package.json

```diff
@@ -6,6 +6,39 @@
   "umd:main": "dist/htm.umd.js",
   "module": "dist/htm.module.js",
   "types": "dist/htm.d.ts",
+  "exports": {
+    ".": {
+      "import": "./dist/htm.mjs",
+      "require": "./dist/htm.js",
+      "browser": "./dist/htm.module.js",
+      "umd": "./dist/htm.umd.js"
```

Comment:
Is this a good opportunity to consider dropping `umd` for the export map? Or perhaps in a separate PR.

I'm not sure I see the value in specifying a `umd` format output in export maps.

Reply:
I was just following the pattern from the main preact repo. I can do the change if needed.

Reply:
No worries, I think this is a good followup item to consider. Not blocking for this PR.

Thanks for doing the work!